include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -288,6 +288,30 @@ class alignment_algorithm :
         {
             res.score = get<3>(cache).score;
         }
+        if constexpr (config_t::template exists<align_cfg::result<with_end_position_type>>())
+        {
+            res.score = get<3>(cache).score;
```

Comment:
At some point, we should have enum values to identify the correct tuple positions.

Reply:
There is already a ticket for refactoring the tuple types into aggregates with named member access, as it is really hard to follow the code with just the tuples.